bzl/deps/BUILD

```diff
@@ -1,11 +0,0 @@
-load("//bzl/rules:platform_transition.bzl", "platform_transition")
-
-package(default_visibility = ["//visibility:public"])
-
-# Build the pico SDK with gcc; it seems to have incompatibilities with g++
-platform_transition(
-    name = "pico_stdlib_c",
-    dep = "@rules_pico//pico:pico_stdlib",
```

Comment:
The new dependency magically grabs the right `gcc` versus `g++` as needed. Not sure how, but I won't complain.